homeassistant/components/airnow/config_flow.py

```diff
@@ -98,6 +98,41 @@ async def async_step_user(self, user_input=None):
             errors=errors,
         )
 
+    @staticmethod
+    @core.callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Create the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle an options flow for AirNow."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input):
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
```

Comment:
My understanding is that `add_suggested_values_to_schema` allows the user to set the value in the `options` dict to `None` if they provide an empty input.  The AirNow API does require the distance parameter to be provided, which is why the initial configuration flow has a default value.  If I use suggested values, I think I still need to have a fallback somewhere to pull the original `config_entry` value rather than relying on the `options` value only per the above conversation?

Reply:
I think on this occasion it doesn't matter so we can skip it.
However instead of just using `int` that it needs to be a number you can consider to use a selector instead so you "force" the user that the number needs to be a certain min or max etc.

Reply:
OK, makes sense.  I think I misunderstood originally how `add_suggested_values_to_schema` was intended to be used.  Should be fixed now.